client/components/data/query-contact-details-cache/index.jsx

```diff
@@ -16,9 +16,16 @@ import { requestContactDetailsCache } from 'state/domains/management/actions';
 
 class QueryContactDetailsCache extends Component {
 	componentWillMount() {
-		if ( this.props.isRequesting || ! isEmpty( this.props.contactDetailsCache ) ) {
+		const {
+			isRequesting,
+			contactDetailsCacheIsEmpty,
```

Comment:
✅ 

Turns out there's a rule for this. I've added [`react/prop-types`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md
) to my local `.eslintrc.js` to try it out and see if it's practical to add to our config.